github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -1,11 +1,12 @@
 presubmits:
   kubevirt/containerized-data-importer:
   - name: pull-cdi-unit-test
+    skip_branches:
+      - release-v1.38
+      - release-v1.43
+      - release-v1.49
```

Comment:
We are specifically not doing this because this is for jobs like unit test, and so far we've been able to use the same job for release and main, the podman change breaks it but it's a one-off